package.json

```diff
@@ -1,7 +1,9 @@
 {
   "scripts": {
     "start": "func host start --verbose",
-    "lint": "eslint 'src/**/*.js' 'migrations/**/*.js' '*.js'"
+    "lint": "eslint 'src/**/*.js' 'migrations/**/*.js' '*.js'",
+    "setupDev": "./scripts/setupDev.js",
```

Comment:
this doesn't work on windows :( 

Should we instead just do `node ./scripts/setupDev.js` since node shebang and windows don't agree ?

Reply:
Thanks for the review and suggestion @AnEmortalKid. I will update this PR shortly